app/internal/wm/os_js.go

```diff
@@ -591,6 +598,21 @@ func (w *window) windowMode(mode WindowMode) {
 	}
 }
 
+func (w *window) orientation(mode OrientationMode) {
+	if j := w.screenOrientation; !j.Truthy() || !j.Get("unlock").Truthy() || !j.Get("lock").Truthy() {
+		return // Browser don't support Screen Orientation API.
+	}
+
+	switch mode {
+	case UnlockedOrientation:
+		w.screenOrientation.Call("unlock")
+	case LockedLandscape:
+		w.screenOrientation.Call("lock", "landscape-primary").Call("then", w.redraw)
```

Comment:
I'm not familiar with the Screen Orientation API, but can we use values that allow both variants (normal and upside down) for each orientation, like for Android?

Reply:
`landscape-primary` = `SCREEN_ORIENTATION_LANDSCAPE`.
`landscape` = `SCREEN_ORIENTATION_USER_LANDSCAPE`.

I update both: JS and Android. Now the user can rotate it. 🎉